### PR TITLE
test(e2e): Add tests for transactions pages

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We want to create snapshots test for each native token transaction page: BTC, ETH, and ICP.

# Changes

- Add additional customizable function to extend `HomepageLoggedIn.waitForReady`
- Create class `TransactionPage` extending `HomepageLoggedIn`.
- Create tests for BTC, ETH and ICP.

# Tests

In this PR we already updated the new screenshots. Please check committed files.

### CODE REVIEW

File: e2e/utils/pages/homepage.page.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ interface HomepageParams {
 	viewportSize?: ViewportSize;
 }
 
-type HomepageLoggedInParams = {
+export type HomepageLoggedInParams = {
```
Comment (DenysKarmazynDFINITY) [inline e2e/utils/pages/homepage.page.ts:25]: Obviously not in this PR, but I guess it's time to extract shared between all pages things from `Homepage` into `Page` class and use it as a parent for all further pages.

**Review by DenysKarmazynDFINITY — APPROVED** [review]
The code LGTM, thanks! I assume not much we can do with the txs call on the ETH page?

**Comment by AntonioVentilii** [comment]
>  I assume not much we can do with the txs call on the ETH page?

 
 I tried to check, but it is really a question of API keys, and for now no, unfort...

File: e2e/utils/pages/homepage.page.ts (hunk 1 above)
Comment (AntonioVentilii) [reply e2e/utils/pages/homepage.page.ts:25]: yes, makes sense! Included in my backlog

**Pull request closed without merging**